### PR TITLE
V3nki/update stream route

### PR DESCRIPTION
**Pull request by venkateshwarv**
Updated API route for stream/historic calls.
Cleaned up some nested templates in methods.

### CODE REVIEW

File: ProfileView/src/[sandbox]/sandbox.ts [hunk 1]
```diff
@@ -88,21 +88,22 @@ export class Sandbox extends LitElement {
             style=${`width: ${this.containerWidth}; height: ${this.containerHeight}; overflow: auto;`}
             class="widget-container"
           >
+            <!-- profile-read-token="so=demoassure&sn=sandbox&ss=profile&sp=r&se=2022-06-17T23:47:34.409Z&sk=sandbox&sig=61BLCJ5+vZtOOvut/7khUQyg0N9KlvbPrJWrYa9lf28=" -->
```
Comment (futurethang) [inline ProfileView/src/[sandbox]/sandbox.ts:91]: Any reason to keep this rather than delete all the way?

File: ProfileView/src/index.ts [hunk 2]
```diff
@@ -49,8 +49,6 @@ export default class CjaasProfileWidget extends LitElement {
   @property({ type: String, attribute: "tape-read-token" }) tapeReadToken:
     | string
     | null = null;
-  @property({ type: String, attribute: "profile-read-token" })
-  profileReadToken: string | null = null;
```
Comment (futurethang) [inline ProfileView/src/index.ts:53]: Breaking change, making this a major version update

File: ProfileView/src/index.ts [hunk 3]
```diff
@@ -384,54 +404,54 @@ export default class CjaasProfileWidget extends LitElement {
...
-                  ? this.renderTimeline(
-                      x.journeyEvents.map((y: any) =>
-                        this.getTimelineItemFromMessage(y)
-                      )
-                    )
-                  : nothing}
-              </md-tab-panel>
-            `;
-          })}
+          ${activityTab} ${tabs.map((x: any) => {})}
```
Comment (futurethang) [inline ProfileView/src/index.ts:426]: What's the vision here? seems we're just stripping away the whole tabs feature, so why leave any code if it's no longer a UI element? As opposed to maybe hiding it under a feature flag attribute?

File: ProfileView/src/index.ts [hunk 4]
```diff
@@ -384,54 +404,54 @@ export default class CjaasProfileWidget extends LitElement {
...
         </md-tabs>
       `;
     } else {
       return activityTab;
     }
   }
 
+  getTab(tab: any) {
```
Comment (futurethang) [inline ProfileView/src/index.ts:434]: I see this replaces . . . however it is not called anywhere. Safe to say this is work in progress?

File: ProfileView/src/index.ts (hunk 3 above)
Comment (venkateshwarv) [reply ProfileView/src/index.ts:426]: Damn. I was moving code around. I should have missed this.

File: ProfileView/src/index.ts (hunk 4 above)
Comment (venkateshwarv) [reply ProfileView/src/index.ts:434]: Let me fix this.

File: ProfileView/src/[sandbox]/sandbox.ts (hunk 1 above)
Comment (venkateshwarv) [reply ProfileView/src/[sandbox]/sandbox.ts:91]: Profile API needs a token with Write access. While we only read data and not write. I need to check with Daniel on this.

Comment (venkateshwarv) [reply ProfileView/src/[sandbox]/sandbox.ts:91]: This Profile API is about to change. We might be needing just a read token.

File: ProfileView/src/index.ts (hunk 3 above)
Comment (venkateshwarv) [reply ProfileView/src/index.ts]: This is fixed now

File: ProfileView/src/index.ts (hunk 2 above)
Comment (venkateshwarv) [reply ProfileView/src/index.ts:53]: Updated ProfileView to 3.0.0

File: ProfileView/src/[sandbox]/sandbox.ts (hunk 1 above)
Comment (venkateshwarv) [reply ProfileView/src/[sandbox]/sandbox.ts:92]: Leaving this comment to stay for now.

File: ProfileView/src/index.ts (hunk 4 above)
Comment (venkateshwarv) [reply ProfileView/src/index.ts:434]: This is fixed

**Review by futurethang — APPROVED** [review]

**Pull request merged**